e2e/playwright/testing-segment-overlays.spec.ts

```diff
@@ -3,18 +3,11 @@ import type { LineInputsType } from '@src/lang/std/sketchcombos'
 import { uuidv4 } from '@src/lib/utils'
 
 import type { EditorFixture } from '@e2e/playwright/fixtures/editorFixture'
-import {
-  deg,
-  getUtils,
-  orRunWhenFullSuiteEnabled,
-  wiggleMove,
-} from '@e2e/playwright/test-utils'
+import { deg, getUtils, wiggleMove } from '@e2e/playwright/test-utils'
 import { expect, test } from '@e2e/playwright/zoo-test'
 
 test.describe('Testing segment overlays', () => {
-  test('Hover over a segment should show its overlay, hovering over the input overlays should show its popover, clicking the input overlay should constrain/unconstrain it:\nfor the following segments', () => {
-    // TODO: fix this test on mac after the electron migration
-    test.fixme(orRunWhenFullSuiteEnabled())
+  test.describe('Hover over a segment should show its overlay, hovering over the input overlays should show its popover, clicking the input overlay should constrain/unconstrain it', () => {
```

Comment:
This is the only functional change in this PR to fix nested `test()` calls not being interpreted correctly by Playwright.